index.html

```diff
@@ -9,7 +9,25 @@
   <!-- Шапка сайта -->
   <header>Шапка сайта</header>
   <!-- Секция 1 -->
-  <section>Секция 1</section>
+  <section class="section" id="warranty">
```

Comment:
You need to replace the id value with "download"

Reply:
changed